demo/secret/common/client.go

```
@@ -5,7 +5,7 @@ import (
 	"flag"
 	"fmt"
 
-	"mosn.io/layotto/sdk/go-sdk/client"
+	"github.com/layotto/go-sdk/client"
```

golangci-lint

File is not `goimports`-ed with -local mosn.io/layotto

(goimports)